swedish/titta-ut/macka/macka.md

```diff
@@ -0,0 +1 @@
+Du går till köket för att göra dig en macka. Väl där hittar du samtliga ingredienser som behövs och äter din nyligen förberedda delikatess i lugn och ro.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.